FIAT/gauss_legendre.py

```diff
@@ -31,3 +36,15 @@ def __init__(self, ref_el, degree):
         dual = GaussLegendreDualSet(ref_el, degree)
         formdegree = ref_el.get_spatial_dimension()  # n-form
         super(GaussLegendre, self).__init__(poly_set, dual, degree, formdegree)
+
+    def tabulate(self, order, points, entity=None):
```

Comment:
Maybe add a comment that this overrides the default with a more numerically stable algorithm?